sections/03_categorize.md

```diff
@@ -142,21 +142,34 @@ This indicates a potential strength of deep methods. It may be possible to
 repurpose features from task to task, improving overall predictions as the field
 tackles new challenges.
 
-Several authors have created reusable feature sets for medical terminologies using
-neural embeddings, as popularized by word2Vec [@tag:Word2Vec]. This approach
-was first used on free text medical notes by De Vine et al.
-[@doi:10.1145/2661829.2661974] with results at or better than traditional methods.
-Y. Choi et al.[@doi:10.1145/2567948.2577348] built embeddings of standardized
-terminologies, such as ICD and NDC, used in widely available administrative
-claims data. By learning terminologies for different entities in the same
-vector space, they can potentially find relationships between different
-domains (e.g. drugs and the diseases they treat). Medical claims data does not
-have the natural document structure of clinical notes, and this issue was
-addressed by E. Choi et al. [@doi:10.1145/2939672.2939823], who built
-embeddings using a multi-layer network architecture which mimics the structure
-of claims data. While promising, difficulties in evaluating the quality of
-these kinds of features and variations in clinical coding practices remain as
-challenges to using them in practice.
+Several authors have created reusable feature sets for medical terminologies
+using natural language processing (NLP) and neural embedding models, as
+popularized by Word2vec [@tag:Word2Vec]. This approach was first used on free
+text medical notes by De Vine et al. [@doi:10.1145/2661829.2661974]
+with results at or better than traditional methods. Y. Choi et al.
+[@doi:10.1145/2567948.2577348] built embeddings of standardized terminologies,
+such as ICD and NDC, used in widely available administrative claims data. By
+learning terminologies for different entities in the same vector space, they can
+potentially find relationships between different domains (e.g. drugs and the
+diseases they treat). Medical claims data does not have the natural document
+structure of clinical notes, and this issue was addressed by E. Choi et al.
+[@doi:10.1145/2939672.2939823], who built embeddings using a multi-layer network
+architecture which mimics the structure of claims data. While promising,
+difficulties in evaluating the quality of these kinds of features and variations
+in clinical coding practices remain as challenges to using them in practice.
+
+NLP has also been applied directly to EHRs to predict disease comorbidities and
+phenotypes as well as novel gene - disease associations. Gligorijevic et al
+[@doi:10.1038/srep32404] built neural embedding models and mined more than 35
+million records collected over a decade. The learned disease representations
+significantly ouperformed state-of-the-art methods, achieving close to 86%
```

Comment:
@enricoferrero  - The issue I have is that I don't think finding these "diseases" have a common SNP (not even genome-wide significance) is interesting or impressive. What would a classifier that just looked for a common word get....? Or a classifier that just picked the closest ICD-9 code by number? The definition of different disease is flawed in my eyes. They looked for two ICD-9 codes with a common SNP, oftentimes these ICD-9 codes are a single disease presenting in a different part of the body. This is why phenotyping algorithms are usually used.

Check out the supplement: https://www.nature.com/article-assets/npg/srep/2016/160831/srep32404/extref/srep32404-s1.csv

 
![image](https://cloud.githubusercontent.com/assets/893692/25484686/04ed5d46-2b29-11e7-8c03-91e762c59e6f.png)

or 

![image](https://cloud.githubusercontent.com/assets/893692/25484717/1860c5c0-2b29-11e7-8614-087c2ce48c47.png)

or

![image](https://cloud.githubusercontent.com/assets/893692/25484721/23f145fe-2b29-11e7-8cec-863f48dd937c.png)

or 

![image](https://cloud.githubusercontent.com/assets/893692/25484735/33a45b94-2b29-11e7-9552-f001e219e319.png)

or 

![image](https://cloud.githubusercontent.com/assets/893692/25484761/443affbc-2b29-11e7-8e0b-5c26742a43b9.png)